health/micro-ui/web/micro-ui-internals/example/src/UICustomizations.js

```
@@ -766,175 +766,176 @@
+             if (e.name === "MP_ACTIONS_EDIT_SETUP") {
+               const key = parseInt(row?.additionalDetails?.key);
+               const resolvedKey = key === 8 ? 7 : key === 9 ? 10 : key || 2;
+               const url = `/${window.contextPath}/employee/microplan/setup-microplan?key=${resolvedKey}&microplanId=${row.id}&campaignId=${row.campaignDetails.id}`;
+               window.location.href = url;
+             }
+             if (e.name == "MP_ACTIONS_VIEW_SUMMARY") {
+               window.location.href = `/${window.contextPath}/employee/microplan/setup-microplan?key=${10}&microplanId=${row.id}&campaignId=${
+                 row.campaignDetails.id
+               }&setup-completed=true`;
+             }
+           };
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)